### DISCUSSION TITLE
Avalonia.Android  11.0.10  [启用修整]Release勾上，发布APK， 使用Assembly.LoadFrom异常

### DISCUSSION THREAD

**Discussion by YSW33** [AvaloniaUI/Avalonia#16190] [Q&A]
### Describe the bug

![img_v3_02c9_ed84a565-449b-47e9-ab11-b167473de3ag](https://github.com/AvaloniaUI/Avalonia/assets/141573398/5bac4a1a-51e0-4915-a025-b5937d7d6d92)

![23824019ef036a7c221b8211d66cb56](https://github.com/AvaloniaUI/Avalonia/assets/141573398/18aab5c4-fa6c-4f36-a809-430cd40ec227)

### To Reproduce

Avalonia.Android  11.0.10  ，发布APK，如果把[启用修整]Release勾上，OverlayDialog.ShowCustomModal使用了Assembly assembly = Assembly.LoadFrom(path)就会报错，提示找不到。

### Avalonia version

11.0.10

### OS

Android

**Comment by hez2010** [comment]
Dynamically loaded assemblies will be removed if you enable trimming. 

Try adding
```
<ItemGroup>
    <TrimmerRootAssembly Include="<your assembly name to be loaded dynamically>" />
</ItemGroup>
```
in your project file.
